test/conformance/kuttl/rangeoperators/standard/policy.yaml

```diff
@@ -3,7 +3,7 @@ kind: ClusterPolicy
 metadata:
   name: check-value
 spec:
-  validationFailureAction: enforce
+  validationFailureAction: Enforce
```

Comment:
@eddycharly is this a current update to Kyverno? using Enforce instead of enforce?

Reply:
Yes, the values of `validationFailureAction` moving forward will be in title case (`Audit` or `Enforce`).